### ISSUE TITLE
UEFI and systemd-boot downwards compatible?

### ISSUE THREAD

**Issue by audiolover** [pop-os/pop#1019]
POP 20.04 this problem: during installation POP does not find my Windows installation on another disk. Various other Linux distro's have no problem finding it and putting it in the bootloader.
Also, the POP bootloader is regularly not recognized by the mainboard bios so the pc will not boot into POP but into Windows instead. Motherboard UEFI version 2.3.1

**Comment by audiolover** [comment]
I have now installed Ubuntu and the pc always boots from the Ubuntu disk. Also, in the boot-menu Windows is listed. Systemd-boot is the problem I think.

**Comment by audiolover** [comment]
More info on the system in use: mainboard: Intel DQ77MK, 16gb RAM, Intel I7 3770, 
NVIDIA GTX 770.

**Comment by Gustav0ar** [comment]
I had the same problem as you about bootloader not finding Windows installation. To solve the issue, you need to copy the EFI from microsoft partition to your popos boot partition. It's something like this:

1. Mount the boot partition from the hdd that has the Windows installation
2. As root, copy Microsoft folder from inside the boot partition to /boot/efi/EFI/ in your PopOS installed disk

That's it, systemd-boot should recognize your Windows in the next boot.

**Comment by Gustav0ar** [comment]
I had the same problem as you about bootloader not finding Windows installation. To solve the issue, you need to copy the EFI from microsoft partition to your popos boot partition. It's something like this:

1. Mount the boot partition from the hdd that has the Windows installation
2. As root, copy Microsoft folder from inside the boot efi partition to /boot/efi/EFI/ in your PopOS installed disk
That's it, systemd-boot should recognize your Windows in the next boot.
If you don't see the boot menu, you need to add a delay like this:

1. Edit as root /boot/efi/loader/loader.conf
2. Add the following statement to a new line in the end of the file: timeout 5

**Comment by audiolover** [comment]
Thank you for your reply. I did this on my previous install of POP but cannot live with the fact that systemd-boot should find the Windows install automatically on installation. Also, the problem that the bios after restart does not recognize the POP bootloader is annoying and is in my opinion also due to systemd-boot.

With other distro's these problems are not present!

**Comment by Gustav0ar** [comment]
Yeah, I'm with you, this should be something that the installation should do for you automatically.
For your problem about your bios not recognizing, check if your BIOS is set for EFI boot...

**Comment by audiolover** [comment]
Bios is set for efi/uefi only.

**Comment by Gustav0ar** [comment]
Hm ok, my last idea: check if secure boot is turned off, it should not be on.

**Comment by audiolover** [comment]
Secure boot is off. Again, all settings the same with my current Ubuntu install. But: always recognized a boot and Windows in the bootloader!
